src/http_server/mod.rs

```diff
@@ -20,6 +21,10 @@ use {
     tap::{Pipe, Tap},
     tower_http::cors::{self, CorsLayer},
     tracing::{info, instrument},
+    wc::geoip::{
+        block::{middleware::GeoBlockLayer, BlockingPolicy},
```

Comment:
`BlockingPolicy` name doesn't have a context to it, we should probably modify `geoip` in a way so it's possible to use it as `geoip::BlockingPolicy` or `geiop::block::Policy`

```suggestion
        block::{middleware::GeoBlockLayer, GeoBlockingPolicy},
```